_remake/lib/init-service-routes.js

```diff
@@ -0,0 +1,198 @@
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const multer = require('multer');
+const extract = require('extract-zip');
+const shell = require('shelljs');
+
+// configure multer storage
+const storage = multer.diskStorage({
+  destination: (req, file, cb) => {
+    cb(null, global.config.location.tmp);
+  },
+  filename: (req, file, cb) => {
+    cb(null, file.originalname);
+  }
+})
+
+const upload = multer({ storage });
+
+// check JWT token received in the header of protected requests
+function checkJWT(req, res, next) {
```

Comment:
I think we should rename this to just `authenticate()` or maybe `authenticateWithJWT`

Reply:
maybe `checkIfAuthenticated`?